pkg/config/config_template.yaml

```diff
@@ -1636,6 +1636,14 @@ api_key:
   #
   # disable_realtime_checks: false
 
+  ## @param infra_agent - object - optional
+  ## Enables Infrastructure Monitoring
```

Comment:
```suggestion
  ## Enables infrastructure monitoring
```

Reply:
Hi! After discussing a bit with Product, we may not want to have that much exposure in template/release notes, so removing those entirely in https://github.com/DataDog/datadog-agent/pull/30137/commits/953c6ba87ede9c0c9d3c6dc6e8a8116db3d50cb3